lens-data-table.html

```diff
@@ -76,119 +77,171 @@
       }
 
     </style>
-    <template if="{{output}}"><div class="row-count">{{output.length}} rows</div></template>
-    <template if="{{pagination}}">
-      <div>
-        <core-icon data-page="{{currentPage | prevPage }}" icon="hardware:keyboard-arrow-left"  on-click="{{movePage}}"></core-icon>
-        Page <input id="pageNum" style="border: 2px solid #eee; padding: 3px; width: 16px;" is="core-input"  committedValue="{{currentPage}}" placeholder="{{currentPage}}" type="text"> of {{numPages}} 
-        <core-icon data-page="{{currentPage | nextPage}}" icon="hardware:keyboard-arrow-right"  on-click="{{movePage}}"></core-icon>
-        <template if="{{startRow && endRow}}">(showing rows {{startRow}} to {{endRow}})</template>
-
-      </div>
-    </template>
-    <div id="table"></div>
-    <template if="{{!readOnly}}">
-    <div>
-      <template if="{{!minimal}}">
-        <template if="{{!firstRowIsHeader}}"> <core-icon icon="check-box-outline-blank" on-click="{{toggleFirstRow}}"></core-icon> </template>
-        <template if="{{firstRowIsHeader}}"> <core-icon icon="check-box" on-click="{{toggleFirstRow}}"></core-icon> </template>
-        First row represents column titles
+
+    <div class="lens-container lens-data">
+      <template if="{{output}}"><div class="row-count">{{output.length}} rows</div></template>
+      <template if="{{pagination}}">
+        <div>
+          <core-icon data-page="{{currentPage | prevPage }}" icon="hardware:keyboard-arrow-left"  on-click="{{movePage}}"></core-icon>
+          Page <input id="pageNum" style="border: 2px solid #eee; padding: 3px; width: 16px;" is="core-input"  committedValue="{{currentPage}}" placeholder="{{currentPage}}" type="text"> of {{numPages}} 
+          <core-icon data-page="{{currentPage | nextPage}}" icon="hardware:keyboard-arrow-right"  on-click="{{movePage}}"></core-icon>
+          <template if="{{startRow && endRow}}">(showing rows {{startRow}} to {{endRow}})</template>
+
+        </div>
       </template>
-    </div> 
-    
-        <template if="{{requireSelection}}">
-          <div id="select" on-click="{{selectOutput}}">Use Selection</div>
-        </template>
+      <div id="table"></div>
+      <template if="{{!readOnly}}">
+        <div>
+          <template if="{{!minimal}}">
+            <template if="{{!firstRowIsHeader}}"> <core-icon icon="check-box-outline-blank" on-click="{{toggleFirstRow}}"></core-icon> </template>
+            <template if="{{firstRowIsHeader}}"> <core-icon icon="check-box" on-click="{{toggleFirstRow}}"></core-icon> </template>
+            First row represents column titles
+
+            <template if="{{!enableSelection}}"> <core-icon icon="check-box-outline-blank" on-click="{{toggleEnableSelection}}"></core-icon> </template>
+            <template if="{{enableSelection}}"> <core-icon icon="check-box" on-click="{{toggleEnableSelection}}"></core-icon> </template>
+            Output selection only
```

Comment:
Let's keep track of the selected data in a separate attr like `selection` and only display this option if a selection is made like we had discussed